lib/board.rb

```diff
@@ -0,0 +1,53 @@
+class Board
+  attr_reader :squares, :wining_combinations
+
+  def initialize
+    @squares = [
+      [{ '1': nil }, { '2': nil }, { '3': nil }],
+      [{ '4': nil }, { '5': nil }, { '6': nil }],
+      [{ '7': nil }, { '8': nil }, { '9': nil }]
+    ]
+
+    @wining_combinations = [
+      [1, 2, 3], [4, 5, 6], [7, 8, 9],
+      [1, 4, 7], [2, 5, 8], [3, 6, 9],
+      [1, 5, 9], [3, 5, 7]
+    ]
```

Comment:
Set it as a constant and as static attribute
Why? Because, it's the same thing for all the Board object.

Reply:
Rubocop is not allowing
C: Style/ClassVars: Replace class var @@WINING_COMBINATIONS with a class instance var.
  @@WINING_COMBINATIONS = [